src/components/_views/ParticipantPickingView.jsx

```diff
@@ -2,31 +2,65 @@ import React from "react";
 import { connect } from "react-redux";
 import SampleComponent from "../SampleComponent";
 import { withRouter } from "react-router";
+import { ParticipantList } from "../ParticipantList";
+import {
+  ADD_PARTICIPANT,
+  GET_PARTICIPANTS
+} from "../../redux/actions/participants";
 
 /**
  * A relatively simple functional component,
  * it's only real purpose is to show that redux works and there's a username that ends up in the store
  */
-function ParticipantPickingView(props) {
-  return (
-    <SampleComponent message="This is the view for choosing participants to groups ">
-      <div>
-        {props.user.name
-          ? `You're logged in as "${props.user.name}"`
-          : `You're not logged in`}
-      </div>
-    </SampleComponent>
-  );
+class ParticipantPickingView extends React.PureComponent {
+  componentDidMount() {
+    const { getParticipants } = this.props;
+    getParticipants();
+  }
+
+  render() {
+    const { participants, user, addParticipant } = this.props;
+
+    return (
+      <SampleComponent>
+        {/* <div>
+          {user.name
+            ? `You're logged in as "${user.name}"`
+            : `You're not logged in`}
+        </div> */}
+        <ParticipantList
+          city="Wroclaw"
+          participants={participants}
+          onAddParticipant={id => {
+            addParticipant(id);
+          }}
+        />
+      </SampleComponent>
+    );
+  }
 }
 
 const mapStateToProps = state => {
   return {
-    user: state.userAuth
+    user: state.userAuth,
+    participants: state.participants
   };
 };
 
 const mapDispatchToProps = dispatch => {
-  return {};
+  return {
+    getParticipants: () => {
+      dispatch({
+        type: GET_PARTICIPANTS
+      });
+    },
+    addParticipant: id => {
+      dispatch({
+        type: ADD_PARTICIPANT,
+        payload: id
+      });
+    }
```

Comment:
While not strictly necessary for things to work, I'd strongly advise in favor of putting action generators in the redux/actions folder. This makes the code more maintainable, when components merely trigger actions and finding their definitions does not require a treasure hunt among all the components.